database/javascript/2017/16065.yaml

```diff
@@ -9,8 +9,6 @@ references:
 affected:
     - name: openssl.js
       version:
-        - "==1.0.2"
-        - "==1.0.1"
+        - "<=0.0.2-security"
```

Comment:
It should be `<=1.1.1'

Reply:
1.1.1 version is published 15 hours ago.

Reply:
Yes.

Reply:
Maybe include that in unsafe versions